### PR TITLE
Display quote from API

### PR DESCRIPTION
**Pull request by touredri**
In this PR we have the following:

- [ ] retrieve data from an API
- [ ] Display the quote
- [ ] Use useState and useEffect hooks

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import './calculator.css';
 import CalculatorKey from './calculatorkeys';
 import calculate from '../logic/calculate';
 import DisplayResult from './display';
+import DisplayQuote from './DisplayQuote';
```
Comment (Shedrack-Sunday) [inline src/components/calculator.js]: - [ ] Based on project requirements, you're meant to call your `DisplayQuote` component directly into the main app component just as the calculator is stand alone. 
- This is asked to test your skills in using component standalone across your application, which is a major beauty of react.

## Requirement:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @touredri     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights 
- Nice file structure for component✔️  
- Good job with the react library modules and methods✔️  
  
# Required Changes  🌵 

- [x] You missed a major point on where to call your new added component.

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).


https://github.com/microverseinc/readme-template

File: src/components/calculator.js (hunk 1 above)
Comment (touredri) [reply src/components/calculator.js]: Ok @Shedrack-Sunday

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @touredri 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Cool structure ✔️ 
- Adherence to changes suggested ✅ 
- Nice UI design 👍

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**